doc/modules/model_evaluation.rst

```diff
@@ -317,6 +317,106 @@ parameter:
     >>> print(cv_results['test_fn'])
     [0 1 2 3 2]
 
+.. _which_scoring_function:
+
+Which scoring function should I use?
+====================================
+
+Before diving into the details of the many scores and metrics, we want
+to give some guidance, inspired by statistical decision theory, on the choice
+of **scoring functions** for **supervised learning**, see [Gneiting2009]_:
+
+  - *Which scoring function should I use?*
+  - *Which scoring function is a good one for my task?*
+
+In a nutshell, if the scoring function is given, e.g. in a kaggle competition
+or in a business context, use that one.
+If you are free to choose, it starts by considering the ultimate goal and application
+of the prediction. It is useful to distinguish two steps:
+
+* Predicting
+* Decision making
+
+**Prediction:**
+Usually, the response variable :math:`Y` is a random variable, in the sense that there
+is *no deterministic* function :math:`Y = g(X)`.
+Instead, there is a probability distribution :math:`F` of :math:`Y`.
+One can aim to predict the whole distribution, known as *probabilistic prediction*,
+or---more the focus of scikit-learn---issue a *point prediction* (or point forecast)
+by choosing a property or functional of that distribution :math:`F`.
+Typical examples are the mean (expected value), the median or a quantile of the
+response variable :math:`Y` (conditionally on :math:`X`).
+
+Once that is settled, use a **strictly consistent** scoring function for that
+target functional, see [Gneiting2009]_.
+For classification **stricly proper scoring rules**, see [Gneiting2007]_, conincide
+with strictly consistent scoring functions.
+One could say that consistent scoring functions act as *truth serum* in that
+they guarantee "that truth telling [. . .] is an optimal strategy in
+expectation" [Gneiting2014]_.
+
+Note that for regressors, the prediction ist done with :term:`predict` while for
```

Comment:
```suggestion
Note that for regressors, the prediction is done with :term:`predict` while for
```

Reply:
Maybe having this sentence before the previous paragraph and close to the sentence mentioning "point prediction ..." would be easier because it connects the naming with the scikit-learn methods.

Reply:
That would break the line of argument in my opinion.